bin/test

```diff
@@ -2,7 +2,7 @@
 set -eo pipefail
 
 # copy the .env template to .env if not there already
-[ ! -f .env ] && cp .env-dist .env
+[ ! -f .env ] || cp .env-dist .env
```

Comment:
That doesn't make sense. No?
The first part says "Does .env NOT exist?"
It should be `&&` meaning, "If it does NOT exist, then copy from .env-dist"